style/scss/_why-contribute.scss

```diff
@@ -50,3 +50,153 @@
     }
   }
 }
+
+@media screen and (min-width: $laptop-size) and (max-width: 1398px){
+  .why-contribute {
+    &__title {
+      padding: 3.5rem 0;
+    }
+
+    &__description {
+      padding: 0 15rem 4rem 15rem;
+      
+      & br {
+        margin-bottom: 0.3rem;
+        content: "A";
+        display: block;
+      }
+    }
+  
+    &__reasons {
+      display: grid;
+      grid-template-columns: repeat(4, 1fr);
+      gap: 3rem;
+
+      &__reason {
+        padding: 0rem;
+  
+        & img {
+          width: 7rem;
+        }
+
+        & p {
+          font-size: 1.9rem;
+          padding-top: 1.4rem;
+        }
+      }
+    }
+  }
+}
+
+@media screen and (min-width: 769px) and (max-width: $laptop-size){
+  .why-contribute {
+    padding: 2rem 0rem;
+  
+    &__title {
+      padding: 2.2rem 0rem;
+    }
+  
+    &__description {
+      padding: 0 10rem 3rem 10rem;
+        & br {
+          margin-bottom: 0.5rem;
+          content: "A";
+          display: block;
+        }
+    }
+  
+    &__reasons {
+      display: grid;
+      grid-template-columns: repeat(6, 1fr);
+      gap: 3rem;
+      padding: 0rem;
+      
+      &__reason {
+        padding: 0.5rem;
+        grid-column: span 2;
+     
+        &:last-child:nth-child(3n - 1) {
+          grid-column-end: -2;
+        }
+        
+        &:nth-last-child(2):nth-child(3n + 1) {
+          grid-column-end: 4;
+        }
+
+        & img {
+          width: 6rem;
+        }
+  
+        & p {
+          padding: 1.5rem 0.3rem 0.3rem 0.3rem;
+        }
+      }
+    }
+  }
+}
+
+@media screen and (min-width: $phone-size) and (max-width: $tablet-size){
+  .why-contribute{
+    gap: 2.4rem;
+    padding: 2.4rem 0;
+
+    &__title {
+      padding: 0rem; 
+      & h2 {
+        padding: 0.8rem 3.95rem;
+        font-size: 2rem;
+        line-height: 113%;
+      }
+
+      & img{
+        margin-top: -1rem;
+        width: 18.3rem;
+        height: 2rem;
+      }
+    }
+
+    &__description {
+      padding: 0 4rem;
+      
+      & p {
+        font-size: 1.5rem;
```

Comment:
The font size should be either 16 or 14. Also, for the sizes smaller than 320px, the font size is 16px, while this one is 15px.

Reply:
done (changed to 1.6rem)